jenkins/pipelines/parallel-execution/Jenkinsfile

```diff
@@ -12,7 +11,7 @@ def uploadToS3() {
       bucket="unknown"
   }
   sh """
-      aws s3 sync ${TESTGRID_HOME}/jobs/ s3://${bucket}/artifacts --exclude "*" --include "*.log" --include "*.properties" --include "*.xml"
+      aws s3 sync ${TESTGRID_HOME}/jobs/${PRODUCT}/builds/ s3://${bucket}/artifacts/jobs/wso2am-intg/builds --include "*"
```

Comment:
'wso2am-intg' is hard-coded. Do we only need to upload builds/ folder?

Reply:
Fixed!